src/content/docs/en/reference/integrations-reference.mdx

```diff
@@ -330,7 +330,7 @@ The address, family and port number supplied by the [NodeJS Net module](https://
 **Why:** To access generated routes and assets for extension (ex. copy content into the generated `/assets` directory). If you plan to transform generated assets, we recommend exploring the [Vite Plugin API](https://vitejs.dev/guide/api-plugin.html) and [configuring via `astro:config:setup`](#updateconfig-option) instead.
 
 ```js
-'astro:build:done'?: (options: { dir: URL; routes: RouteData[] }) => void | Promise<void>;
+'astro:build:done'?: (options: { dir: URL; routes: RouteData[], pages: PageData[] }) => void | Promise<void>;
```

Comment:
`pages` here should probably be typed as `{ pathname: string }>[]` here to match Astro's types, but it'd be a great idea for us to add a `PageData` type into Astro's type definitions to make this more future proof!

Reply:
Ah yep, I missed this when removing the types. I'll fix this in a sec!